src/ResourceManager/DataFactoryV2/Commands.DataFactoryV2/help/Remove-AzureRmDataFactoryV2IntegrationRuntime.md

```diff
@@ -14,21 +14,22 @@ Removes an integration runtime.
 
 ### ByIntegrationRuntimeName (Default)
```

Comment:
Please regenerate these help files after making changes to parameter sets.

Reply:
Done